scripts/mocha.js

```diff
@@ -0,0 +1,30 @@
+const Mocha = require('mocha');
+const path = require('path');
+const express = require('express');
```

Comment:
Might not be a big problem, but a possible optimization for the build time and dependencies would be to have a simple server giving the static [static files with core modules](https://gist.github.com/amejiarosario/53afae82e18db30dadc9bc39035778e5). This way 28 dependencies of express go away. Or another simpler implementation with [http-server](https://www.npmjs.com/package/http-server)